drivers/console/esp32_hw_cdc_console.c

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (c) 2021 Espressif Systems (Shanghai) Co., Ltd.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <kernel.h>
+#include <sys/printk.h>
+#include <device.h>
+#include <init.h>
+#include <soc.h>
+
+extern void __stdout_hook_install(int (*hook)(int));
+extern void __printk_hook_install(int (*fn)(int));
+
+static int esp32_hw_console_out(int character)
+{
+	/* console over cdc on esp32 uses the same uart rom
+	 * functions
+	 */
+	esp_rom_usb_uart_tx_one_char(character);
```

Comment:
I do not like to see `_usb_` in identifiers that are not related to USB subsystem in Zephyr.